### PR TITLE
Add Fig.app latest

### PR DESCRIPTION
**Pull request by mschrage**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

**Comment by mschrage** [comment]
cc @sullivan-sean @brendanfalk

File: Casks/fig.rb [hunk 1]
```diff
@@ -2,7 +2,7 @@
   version "1.0.49,270"
   sha256 :no_check
 
-  url "http://versions.withfig.com/fig@latest.dmg",
+  url "https://versions.withfig.com/fig@latest.dmg",
```
Comment (bevanjkay) [inline Casks/fig.rb:5]: Is there a reason why we shouldn't use the versioned download here?

```suggestion
    url "https://versions.withfig.com/fig%20#{version.before_comma}.dmg",
```

Would require adding the `sha256` value also.

File: Casks/fig.rb [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    strategy :sparkle
+  end
+
+  depends_on macos: ">= :mojave"
+
+  app "Fig.app"
+
+  uninstall script:
+                        {
+                          executable: "~/.fig/tools/uninstall-script.sh",
```
Comment (bevanjkay) [inline Casks/fig.rb:22]: Is the script bundled with the app the same as what is moved to the `.fig` directory.
After install the `.fig` directory isn't available straight away, so the below may work?

```suggestion
                          executable: "/Applications/Fig.app/Contents/Resources/uninstall.sh",
```

**Comment by miccal** [comment]
Thank you @mschrage.

**Pull request merged**